support-frontend/assets/helpers/tracking/trackingOphan.ts

```diff
@@ -136,40 +127,9 @@ const setReferrerDataInLocalStorage = (
 
 const getPageViewId = (): string => ophan.viewId;
 
-/**
- * To be used when navigating with React Router
- * in order to generate new Page View event in Ophan
- */
-const navigateWithPageView = (
```

Comment:
Would we need this if we go back to navigating with React Router? @GHaberis

Reply:
Hypothetically it might be useful in the future, but unless we're using this I don't think we need to keep it tbh.